src/utils/provideLocalStorageDataClass.ts

```diff
@@ -3,6 +3,7 @@ import { pseudoRandom32CharHex } from './pseudoRandom32CharHex'
 import { orderBy } from 'lodash-es'
 import { ensureString } from './ensureString'
 import { DataClassAttributes } from '../Data/types'
+import { scrivitoTenantId } from '../config/scrivitoTenants'
```

Comment:
* [ ] The `config` package already has a dependency on `utils`. Let's not introduce the other direction and move `scrivitoTenants` to utils.

Reply:
I see you point. But I don't really think, that `scrivitoTenants` is a good candidate for `utils`. I moved `provideLocalStorageDataClass` instead to `src/Data` to avoid the circular package dependency.